### PR TITLE
concord: init at 2.2.0

### PR DESCRIPTION
**Pull request by transgirllucy**
###### Description of changes

I made it compatible with Nix

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/libraries/concord/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{ lib, stdenv, fetchFromGitHub, cmake, pkgs }:
```
Comment (wegank) [inline pkgs/development/libraries/concord/default.nix:1]:
```suggestion
{ lib, stdenv, fetchFromGitHub, cmake, curl }:
```

File: pkgs/development/libraries/concord/default.nix [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  version = "2.2.0";
+
+  src = fetchFromGitHub {
+    owner = "cogmasters";
+    repo = "concord";
+    rev = "v${version}";
+    sha256 = "sha256-+nuVyqA/V/J6DWAeIs9Pv90ry3px1gJsF460qvfSEH8=";
+  };
+
+  buildInputs = [ pkgs.curl ];
```
Comment (wegank) [inline pkgs/development/libraries/concord/default.nix:14]:
```suggestion
  buildInputs = [ curl ];
```

File: pkgs/development/libraries/concord/default.nix [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+stdenv.mkDerivation rec {
+  pname = "concord";
+  version = "2.2.0";
+
+  src = fetchFromGitHub {
+    owner = "cogmasters";
+    repo = "concord";
+    rev = "v${version}";
+    sha256 = "sha256-+nuVyqA/V/J6DWAeIs9Pv90ry3px1gJsF460qvfSEH8=";
```
Comment (wegank) [inline pkgs/development/libraries/concord/default.nix:11]:
```suggestion
    hash = "sha256-+nuVyqA/V/J6DWAeIs9Pv90ry3px1gJsF460qvfSEH8=";
```

File: pkgs/development/libraries/concord/default.nix [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  makeFlags = [ "PREFIX=$(out)/local"];
+
+  meta = with lib; {
+    home = "https://cogmasters.github.io/concord/";
+    description = "A Discord API wrapper library made in C ";
+    license = licenses.mit;
+    maintainers = with maintainers; [ mrtuxa ];
+    platforms = platforms.linux;
+  };
+
```
Comment (wegank) [inline pkgs/development/libraries/concord/default.nix:25]: I guess you need to solve this manually. Also, please squash the commits with the message in the title.

```suggestion
  };
 
```

Comment (wegank) [reply pkgs/development/libraries/concord/default.nix:25]: Not resolved.

**Review by wegank — CHANGES_REQUESTED** [review]
Capture d’écran 2023-03-29 à 17 22 25

**Comment by mrtuxa** [comment]
> Capture d’écran 2023-03-29 à 17 22 25

how can i refactor it with a tool?

**Comment by wegank** [comment]
I suggest you to proceed in command line with git (or any other equivalent), without using the GitHub interface.

**Pull request closed without merging**